build.gradle.kts

```diff
@@ -402,6 +397,21 @@ allprojects {
                 java {
                     targetExclude(*javaccGeneratedPatterns + "**/test/java/*.java")
                     licenseHeaderFile(licenseHeaderFile)
+                    if (!project.props.bool("junit4", default = false)) {
+                        replace("junit5: Test", "org.junit.Test", "org.junit.jupiter.api.Test")
```

Comment:
Is string/replace still necessary if most of the tests are migrated already ?

Reply:
I think we can drop that later.

The use case is as follows:
1) You drop `junit4=true` from `mongodb/gradle.properties`, and execute `./gradlew spotlessApply`. It would use the replacements to automatically replace the bits in `mongodb` module.

2) When somebody implemented JUnit4-based test, then they rebase on top of master.
The replacement rules would convert the code to the new format automatically!
That is nice, isn't it?

Reply:
Unfortunately, `assertEquals`, `assertTrue`, and frields have different APIs between 4 and 5, so they can't be converted with a regexp :-(